pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -184,14 +184,39 @@ func (s configSource) String() string {
 type configProcessingRetval int
 
 const (
-	configOK        configProcessingRetval = iota
-	configReqFailed                        // failed to request latest config
-	obsoleteConfig                         // newer config is already applied
-	invalidConfig                          // config is not valid (cannot be parsed, UUID mismatch, bad signature, etc.)
-	skipConfig                             // reboot or shutdown flag is set
-	defferConfig                           // not ready to process config yet
+	configOK         configProcessingRetval = iota
+	configReqFailed                         // failed to request latest config
+	obsoleteConfig                          // newer config is already applied
+	invalidConfig                           // config is not valid (cannot be parsed, UUID mismatch, bad signature, etc.)
+	skipConfigReboot                        // reboot or shutdown flag is set
+	skipConfigUpdate                        // update flag is set
+	defferConfig                            // not ready to process config yet
```

Comment:
I saw this typo earlier and ignored it since not your PR. But ince you end up creating more copies of it in String() would you mind fixing it to be spelled "defer".

Reply:
Yep, sure, I will do that now.